crates/eigen-cli/src/generate.rs

```diff
@@ -0,0 +1,169 @@
+use crate::args::KeyType;
+use crate::EigenKeyCliError;
+use ark_ff::UniformRand;
+use ark_serialize::{CanonicalSerialize, SerializationError};
+use eth_keystore::encrypt_key;
+use rand::{distributions::Alphanumeric, Rng};
+use rand_core::OsRng;
+use std::io::Write;
+use std::{
+    fs::{self, File},
+    path::Path,
+};
+use uuid::Uuid;
+
+const PASSWORD_LENGTH: usize = 20;
```

Comment:
why so short?

Reply:
We took this constant from the Go implementation, and since this keys are for testing purposes we thought it's fine. Should we extend it?

Reply:
ah, k. let's keep it the same then. thanks!